### PR TITLE
Example of fluent builders.

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

**Comment by brendandburns** [comment]
@iocanel any chance you can give a quick review?

Thanks!

**Comment by iocanel** [comment]
Sure

File: kubernetes/pom.xml [hunk 1]
```diff
@@ -200,7 +200,7 @@
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <swagger-core-version>1.5.12</swagger-core-version>
-    <sundrio.version>0.8.0</sundrio.version>
+    <sundrio.version>0.9.2</sundrio.version>
```
Comment (iocanel) [inline kubernetes/pom.xml]: FWIW, this also addresses an issue with efficiency when it comes to validation. +1

**Comment by iocanel** [comment]
This looks great as is, but I'd like to propose a future improvement:
It would be really valuable if we also included an example of Visitors to perform modifications n or more resources. For example:

- To add a label to all resources of a kubernetes list.
- To update the version of an image.
- To add a volume to multiple containers.

I can send a followup PR.

**Comment by brendandburns** [comment]
Cool.  @iocanel if you can reply with '/lgtm' then the automation will add the appropriate label so that this can merge.

Then a PR with even more examples would be awesome.

Thanks!

**Comment by brendandburns** [comment]
@iocanel I sent you an invite to be a member of this repo so that I can assign you things so you can lgtm...

*sigh*

I'm going to manually stamp this one to make progress.

**Comment by brendandburns** [comment]
self-lgtm to work around bureacracy...

**Pull request merged**